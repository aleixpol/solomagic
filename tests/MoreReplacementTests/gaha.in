\ref something0
\mt somethingsomething gaha somethingsomething

\ref something1
\mt somethingsomething ragaha somethingsomething

