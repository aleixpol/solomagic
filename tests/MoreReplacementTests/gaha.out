\ref something0
\mt somethingsomething gaha somethingsomething
\ma somethingsomething gaaha somethingsomething

\ref something1
\mt somethingsomething ragaha somethingsomething
\ma somethingsomething ragaha somethingsomething
